lib/u3d/commands_generator.rb

```diff
@@ -148,6 +148,15 @@ def run
         end
       end
 
+      command :dependencies do |c|
+        c.syntax = 'u3d dependencies [-p | --package_manager <package manager>]'
+        c.description = 'Installs Unity dependencies. [Linux only]'
```

Comment:
summary + description. We can be more verbose on the description part. In particular
*  detail how it works for detecting the package_manager, etc.
* link to Linux forum post
* explain how we deal with things like `debconf (>= 0.5) | debconf-2.0`
* the fact we only implement the default deps, and not the ones for platforms (WebGL, etc)

Reply:
Ok